recipe/meta.yaml

```
@@ -1,34 +1,35 @@
-{% set version = "2.16.3" %}
+{% set version = "2.14.6" %}
 
 package:
   name: pydantic-core
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/pydantic-core/pydantic_core-{{ version }}.tar.gz
-  sha256: 1cac689f80a3abab2d3c0048b29eea5751114054f032a941a32de4c852c59cad
+  sha256: 1fd0c1d395372843fba13a51c28e3bb9d59bd7aebfeb17358ffaaa1e4dbbe948
 
 build:
+  script_env:
+    # on osx-64, we get an error from install_name_tool: changing install names or rpaths can't be redone for: <path> (for architecture x86_64) because larger updated load commands do not fit (the program must be relinked, and you may need to use -headerpad or -headerpad_max_install_names)
+    # After some investigation, rustc doesn't seem to read our LDFLAGS
+    - RUSTFLAGS=-C link-args=-headerpad_max_install_names  # [osx and x86_64]
   script:
-    # PyPy has weird sysconfigdata name
-    - rm -f $PREFIX/lib/pypy$PY_VER/_sysconfigdata.py  # [build_platform != target_platform and target_platform == "linux-ppc64le"]
-    - {{ PYTHON }} -m pip install . -vv
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
     - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
   number: 0
+  missing_dso_whitelist:  # [s390x]
+    - $RPATH/ld64.so.1    # [s390x]
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - maturin >=1,<2                         # [build_platform != target_platform]
     - {{ compiler("c") }}
     - {{ compiler("rust") }}
     - cargo-bundle-licenses
   host:
     - pip
     - python
-    - maturin >=1,<2
-    - typing-extensions  >=4.6.0,!=4.7.0
+    - maturin 1.3.1
+    - typing-extensions  4.9.0
```

```suggestion
    - maturin >=1,<2
    - typing-extensions  >=4.6.0,!=4.7.0
```
Adding hard/strict pin will just make our life harder in the future. You can keep the upstream pins.